graylog2-web-interface/src/views/components/aggregationbuilder/AggregationControls.jsx

```diff
@@ -196,10 +197,12 @@ export default class AggregationControls extends React.Component<Props, State> {
               </DescriptionBox>
             )}
             {VisualizationConfigType && (
+              /* eslint-disable react/jsx-pascal-case */
               <DescriptionBox description="Visualization config" help="Configuration specifically for the selected visualization type.">
                 <VisualizationConfigType.component onChange={this._onVisualizationConfigChange}
                                                    config={visualizationConfig} />
               </DescriptionBox>
+              /* eslint-enable react/jsx-pascal-case */
```

Comment:
We could do something like
```
const VisualizationConfig = VisualizationConfigType?.component;
...
{VisualizationConfig && (
  ...
   <VisualizationConfig ... />
  ...
)}
```

To avoid `eslint-disable`.